packages/react-components/tests/systems-summary/main-menu-task-state.test.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { MainMenuTaskState } from '../../lib';
+import { render } from '@testing-library/react';
+import { tasks } from './test.utils';
+
+test('smoke test', () => {
```

Comment:
Needs tests on the `tasks` props, check that the correctly number of tasks is render for each category.